src/web-ui/src/public/ProductDetail.vue

```diff
@@ -157,77 +74,22 @@ export default {
     },
   },
   methods: {
-    ...mapActions(['addToCart']),
-    resetQuantity() {
-      this.quantity = 1;
-    },
-    async addProductToCart() {
-      await this.addToCart({
-        product: {
-          ...this.product,
-          price: this.discount ? discountProductPrice(this.product.price) : this.product.price,
-        },
-        quantity: this.quantity,
-        feature: this.$route.query.feature,
-        exp: this.$route.query.exp,
-      });
-
-      this.renderAddedToCartConfirmation();
-
-      this.resetQuantity();
-      AnalyticsHandler.recordShoppingCart(this.user, this.cart)
-    },
     async fetchData() {
       await this.getProductByID(this.$route.params.id);
 
       this.getRelatedProducts();
-
-      this.recordProductViewed(this.$route.query.feature, this.$route.query.exp, this.$route.query.di);
```

Comment:
Won't we need this to record the product interaction on personalize?

Reply:
As discussed, we will be recording interactions on click.